playbooks/group_vars/marist.yml

```diff
@@ -26,3 +26,5 @@ zowe_smpe_volser: ZOWE02
 zowe_zlux_terminal_telnet_port: 623
 # optional, this is the folder where we store Zowe FMIDs on the z/OS server
 zowe_fmids_dir_remote: /ZOWE/zowe-fmids
+# Restrict permissions to keystores after the certificate setup script
+zowe_lock_keystore: false
```

Comment:
sure, I was not certain where to put them, is it now correct?

Reply:
It sounds like Jack was suggesting the value defaults to `true` in `playbooks/roles/configure/defaults/main.yml`. Do you need it set to false for some specific use cases? Could we set it in the test for that?

Reply:
This have to be false as default in playbooks. We need to add CA certificate to apiml truststore for testing x509 authentication.

Reply:
So, Jack's point is that we want to be careful about trying to test as close to the customer's defaults as we can, which means the default should be true. I'm suggesting that we set it to false for the specific tests that are going to be checking for client certificates, which I guess are in a subsequent PR? Why does this not work?

Reply:
@jackjia-ibm - for your reference I had a chat with Andrej and David and got a better understanding. The current way that it is set-up/planned is that `zowe_lock_keystore` is set to `false` for all automation as it is required for the configuration of client certificate support. The test plan for that is to add a new test that runs in all of the suites (convenience, ptf, keyring etc) and check client certificates, so the setting would be required to `false` on everything. My understanding that there was going to be a new extended test suite for client certificate support wasn't correct, which might have enabled just setting `zowe_lock_keystore: false` for that suite.
Due to this I think we have to have `zowe_lock_keystore: false` in all the automated tests otherwise it won't work, so setting this in the defaults/main.yml is probably the least confusing place.
This PR is holding up other work, so I'll merge it now, but please let me/us know if we've missed anything here and we can spin it into the next/a. new PR.

Reply:
After getting Jack's opinion and learning about how the installation and sanity tests are designed to act independently. 
Summary is:
- playbooks provide result exactly same as manual installation with possibilities for customization — same as what we suggested in docs.zowe.org
- installation test defines the test case. in our case, this test case should be customize the installation to use x509 client certificate and the corresponding test cases should pass
- sanity test should be able to test any zowe instance, no matter it’s installation by playbooks or not
(see slack convo for more details)

I think that gives us the following options:

1. No new extended test suite:
- Setting `'zowe_lock_keystore': false` in all the installation test scripts `tests/installation/src/__tests__ `
- A new toggle something like `CLIENT_CERTIFICATE_CONFIGURED` set to `true` in `playbooks/roles/verify/tasks/main.yml`

2. 1+ extended test suite for client certificates
- Setting `'zowe_lock_keystore': false` in the specific test suite(s)
- A new toggle something like CLIENT_CERTIFICATE_CONFIGURED set to `not zowe_lock_keystore` in `playbooks/roles/verify/tasks/main.yml`

Both cases:
- `zowe_lock_keystore: true` in `playbooks/roles/configure/defaults/main.yml`
- When the new test is added, it only runs if `CLIENT_CERTIFICATE_CONFIGURED` is defined and `true`